sql/src/test/java/org/apache/druid/sql/calcite/schema/DruidSchemaTest.java

```diff
@@ -309,7 +323,7 @@ public void testNullDatasource() throws IOException
                                                        .stream()
                                                        .map(AvailableSegmentMetadata::getSegment)
                                                        .collect(Collectors.toList());
-    Assert.assertEquals(segments.size(), 3);
+    Assert.assertEquals(segments.size(), 4);
```

Comment:
If 4 is the expected value, it should be the first argument. (Similar comment for other occurrences below).

Reply:
fixed here and other places.